erb_transformer.rb

```diff
@@ -1,20 +1,24 @@
-delimiter = ARGV[0]
-engine = ARGV[1]
-handler = case engine
-  when 'erubi'
-    require 'erubi'
-    Erubi::Engine
-  when 'erubis'
-    require 'erubis'
-    Erubis::Eruby
-  when 'erb'
-    require 'erb'
-    ERB
-  else raise "Unknown templating engine `#{engine}`"
-end
+begin
+  delimiter = ARGV[0]
+  engine = ARGV[1]
+  handler = case engine
+    when 'erubi'
+      require 'erubi'
+      Erubi::Engine
+    when 'erubis'
+      require 'erubis'
+      Erubis::Eruby
+    when 'erb'
+      require 'erb'
+      ERB
+    else raise "Unknown templating engine `#{engine}`"
+  end
 
-if engine == 'erubi'
-  puts "#{delimiter}#{eval(handler.new(STDIN.read).src)}#{delimiter}"
-else
-  puts "#{delimiter}#{handler.new(STDIN.read).result}#{delimiter}"
-end
+  if engine == 'erubi'
+    puts "#{delimiter}#{eval(handler.new(STDIN.read).src)}#{delimiter}"
+  else
+    puts "#{delimiter}#{handler.new(STDIN.read).result}#{delimiter}"
+  end
+rescue => error
```

Comment:
Why is this required in "rewrite to TS"?

Reply:
Hey, I can't remember now. I probably just fixed this in testing as an attempt to capture error output.

Reply:
oh yeah, I think this was to fix #63.